src/sentry/tasks/post_process.py

```diff
@@ -525,6 +532,8 @@ def post_process_group(
                 return
             with metrics.timer("tasks.post_process.delete_event_cache"):
                 event_processing_store.delete_by_key(cache_key)
+                if options.get("rc-processing-split-write-double"):
+                    transactions_processing_store.delete_by_key(cache_key)
```

Comment:
If we have to turn this option off, we'll strand some data in the new redis-cluster. As the data would be set, but not removed. That shouldn't matter too much, but is possible.